### ISSUE TITLE
Accessibility mini survey

### ISSUE THREAD

**Issue by castaway** [UKHackspaceFoundation/website#88]
Each space should be able to edit/display a copy of a mini accessibility (finance, physical, psychically etc) survey for their space, based on Damian's sketch: https://docs.google.com/drawings/d/1y5PckzOhzen42qmRTY7qi9GrOM8pBbEX66bFH-yPumU/edit?usp=sharing
